apps/electron/scripts/generate-assets.mjs

```diff
@@ -19,9 +20,20 @@ console.log('build with following dir', {
   publicAffineOutDir,
 });
 
+// step 0: check version match
+const electronPackageJson = await fs.readFile(
```

Comment:
`import()` it?

Reply:
use import will cause ERR_IMPORT_ASSERTION_TYPE_MISSING error, can we set type=module in package.json?

Reply:
done